src/main/java/com/denizenscript/depenizen/bukkit/objects/mythicmobs/MythicMobsMobTag.java

```diff
@@ -299,7 +316,7 @@ public static void registerTags() {
         // @plugin Depenizen, MythicMobs
         // @description
         // Returns the MythicMob's threat table, only containing players.
-        // Map is in the formatting of "UUID/Threat|UUID/Threat"
+        // The map's key is the player's UUID, and the value is the amount of threat.
```

Comment:
Generally phrased like `A map of player's UUIDs to the threat amount` or something like that